drivers/jtag/jtag-aspeed.c

```diff
@@ -71,7 +71,7 @@
 #define ASPEED_JTAG_SW_MODE_TDIO		BIT(16)
 
 /* ASPEED_JTAG_TCK : TCK Control */
-#define ASPEED_JTAG_TCK_DIVISOR_MASK	GENMASK(11, 0)
+#define ASPEED_JTAG_TCK_DIVISOR_MASK	GENMASK(10, 0)
```

Comment:
Aspeed used https://github.com/AspeedTech-BMC/linux/commit/dbd009c011d14214b4d5379abf5b5a3da55769c3
 to The bit fields of clock divisor in the JTAG HW mode 2 is 11:0 instead of 10:0. Are you reverting these chnages?

Reply:
It works with GENMASK(10,0)
will check with Aspeed after release of V02C